src/main/kubectl.ts

```diff
@@ -123,6 +123,10 @@ export class Kubectl {
   }
 
   public async getPath(bundled = false): Promise<string> {
+    if (bundled) {
```

Comment:
Is this `if (bundled) {}` related to this PR?

Reply:
Sort of, the `bundled` variable is unused and I was unsure of how what to do. I checked all the uses of this method and none of them pass in any parameters.